## Quiz #0101

#### Provide the answers in Python code:

1).  In the class A, students' heights are:

161.5 <br>
165.9 <br>
168.7 <br>
163.1 <br>
170.8 <br>
167.2 <br>

Calculate the mean height by using Python 'list' and 'for' loop:

In [3]:
A=[161.5,165.9,168.7,163.1,170.8,167.2]

sum=0
for i in A:
  sum +=i

result=sum/len(A)

print(result)

166.20000000000002


2).  In the class B, students' heights are:

172.5 <br>
162.9 <br>
161.7 <br>
160.1 <br>
161.8 <br>
169.2 <br>

Calculate the mean height by using Python 'list' and 'for' loop:

In [4]:
B=[172.5,162.9,161.7,160.1,161.8,169.2]

sum=0
for i in B:
  sum +=i

result=sum/len(B)

print(result)

164.70000000000002


3). The class A and the class B are joined together to form the class AB. Calculate the mean height by using Python 'list' and 'for' loop:

In [7]:
AB=A+B

sum=0
for i in AB:
  sum +=i

result=sum/len(AB)

print(result)

165.45000000000002


4). Sort the heights of the class AB.

In [8]:
print(AB)
AB.sort()
print(AB)

[161.5, 165.9, 168.7, 163.1, 170.8, 167.2, 172.5, 162.9, 161.7, 160.1, 161.8, 169.2]
[160.1, 161.5, 161.7, 161.8, 162.9, 163.1, 165.9, 167.2, 168.7, 169.2, 170.8, 172.5]


5). The class C is formed with students with the height less than 165. Make a list 'C' containing the heights of the class C.

In [9]:
C=[]
for i in AB:
  if i <165:
    C.append(i)

print(C)

[160.1, 161.5, 161.7, 161.8, 162.9, 163.1]


6). The class D is formed with students with the height larger than 160 and less than 170. Make a list containing the heights of the class D.

In [10]:
D=[]
for i in AB:
  if i > 165 and i < 170:
    D.append(i)

print(D)

[165.9, 167.2, 168.7, 169.2]
